Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "asdf-core-rust"
+version = "0.1.0"
+edition = "2021"
+
+description = "Extendable version manager with support for Ruby, Node.js, Elixir, Erlang & more - now in Rust!"
+homepage = "https://asdf-vm.com/"
+license = "MIT"
+keywords = ["version-manager", "asdf-vm", "multiple-languages", "shell"]
+categories = ["command-line-interface", "command-line-utilities", "config", "development-tools"]
```

Comment:
I also added these for good measure, but I think they are only used by crates.io, which I'm not sure we will be using.